src/sentry/static/sentry/app/views/settings/organizationRelays/modals/handleXhrErrorResponse.tsx

```diff
@@ -0,0 +1,71 @@
+import {t} from 'app/locale';
+
+type Error = {
+  type:
+    | 'unknown'
+    | 'bad-structure'
+    | 'missing-name'
+    | 'empty-name'
+    | 'missing-key'
+    | 'invalid-key';
+  message: string;
+};
+
+type XhrError = {
+  responseJSON?: {
+    trustedRelays: Array<string>;
+  };
+};
+
+function handleError(error: XhrError): Error {
+  const errorMessage = error.responseJSON?.trustedRelays[0];
+
+  if (!errorMessage) {
+    return {
+      type: 'unknown',
+      message: t('An unknown error occurred while saving relay public key'),
```

Comment:
we have more specific error messages, for example, if the user enters an invalid key, the error "invalid key" will be shown. 

I would also like to return a more abstract error message, as it can be anything ... it can be a server problem, etc...not necessarily an error generated by the user